mlrun/projects/pipelines.py

```diff
@@ -118,19 +119,16 @@ def clear_tmp(self):
 class FunctionsDict:
     """Virtual dictionary hosting the project functions, cached or in the DB"""
 
-    def __init__(self, project, decorator=None, disable_auto_mount=False):
+    def __init__(self, project, decorator=None):
         self.project = project
         self._decorator = decorator
-        self._disable_auto_mount = disable_auto_mount
```

Comment:
Why ?

Reply:
`disable_auto_mount` sould be a func spec attr vs a workflow run param, you would like to disable auto mound when you have alternative set of mount cred (which should be set at the func level)